DoSomethingArchive/forge
# Updating the social media icons available on Forge.

#### What's this PR do?

This PR updates the available social media icons in Forge along with updating the pattern library.

It also organizes the list of social media icon classes and the examples in the pattern library by alphabetical order because I'm a crazy person. tee hee 👯 
#### Where should the reviewer start?

Just pass some eyes on the codez 👀 
#### How should this be manually tested?

Pull down, set up the pattern library and make sure you see the new and updated icons.
#### What are the relevant tickets?

Fixes #557 
#### Screenshots

These are the new/updated icons!

<img width="243" alt="screen shot 2016-06-21 at 12 18 07 pm" src="https://cloud.githubusercontent.com/assets/105849/16243223/030a3b7e-37ab-11e6-9931-bb95ae82fcdb.png">

<img width="251" alt="screen shot 2016-06-21 at 12 18 21 pm" src="https://cloud.githubusercontent.com/assets/105849/16243224/031ab3aa-37ab-11e6-9549-2f1a67d66b70.png">

<img width="245" alt="screen shot 2016-06-21 at 12 18 30 pm" src="https://cloud.githubusercontent.com/assets/105849/16243225/0328ffb4-37ab-11e6-981d-782136fb5185.png">

---

@DoSomething/front-end
